go.mod

```diff
@@ -139,8 +139,9 @@ require (
 	github.com/rancher/remotedialer v0.4.0
 	github.com/rancher/rke v1.6.1
 	github.com/rancher/steve v0.0.0-20240806133920-61be17faa3d2
-	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20210727200656-10b094e30007
+	github.com/rancher/system-upgrade-controller/pkg/apis v0.0.0-20240301001845-4eacc2dabbde
 	github.com/rancher/wrangler v1.1.2
+	github.com/rancher/wrangler/v2 v2.2.0-rc6
```

Comment:
Remove this and just wait till https://github.com/rancher/rancher/pull/46950 is merged and re-run CI.